R/obic.R

```diff
@@ -193,6 +207,14 @@ obic_field <- function(B_SOILTYPE_AGR,B_GWL_CLASS,B_SC_WENR,B_HELP_WENR,B_AER_CB
                    M_MECHWEEDS = M_MECHWEEDS,
                    M_PESTICIDES_DST = M_PESTICIDES_DST)
   
+  # add B_AREA_DROUGHT
+  dt$B_AREA_DROUGHT <- B_AREA_DROUGHT
+  if(any(is.na(B_AREA_DROUGHT) & useClassicOBI == FALSE)){
+    dt[is.na(B_AREA_DROUGHT), B_AREA_DROUGHT := fifelse(B_SOILTYPE_AGR %in% c('dekzand', 'dalgrond', 'loess', 'duinzand') & 
+                                                          B_AER_CBS %in% c('LG14', 'Zuid-Limburg', 'LG13', 'Zuidelijk Veehouderijgebied', 'LG12', 'Zuidwest-Brabant'),
+                                                        TRUE, FALSE)]
+    warning('B_AREA_DROUGHT has missing values, a default value based on soil type will be used.')
```

Comment:
remove the warnings (when running in production, these will continuously be generated). @SvenVw is that needed?

Reply:
The user or service should provide a value for a b_area_drought. If no value is provided an error should be generated similiar to other variables. Otherwise you have to implement filling missing values for other input arguments as well

Reply:
brent added indeed a default when the variable was missing. So in that case its oke to remove the warning.